utils/build/docker/nodejs/express4/app.js

```diff
@@ -44,7 +44,8 @@ app.get('/healthcheck', (req, res) => {
     library: {
       language: 'nodejs',
       version: require('dd-trace/package.json').version,
-      libddwaf_version: require('@datadog/native-appsec/package.json').libddwaf_version
+      libddwaf_version: require('dd-trace/node_modules/@datadog/native-appsec/package.json').libddwaf_version,
```

Comment:
why change from the direct require ?

Reply:
That didn't work, but reverting doesn't work either. I'm looking into it.

Reply:
I misunderstood the question actually, the reason is because when dd-trace is linked the `@datadog/native-appsec` is not flattened at the root, so we need to require it from under `dd-trace`. This is also why the `.npmrc` files were added with a `nested` configuration.